src/styles/index.scss

```diff
@@ -1,3 +1,58 @@
+@import './utils/variables';
+@import 'block/header';
+@import 'block/pruduct-card';
+@import 'block/starts';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (max-width: 487px) {
+  .main {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
```

Comment:
According to the code style rules, there should be a space after `repeat` function: `repeat(2, 200px)` should be `repeat(2 , 200px)`.